ETC.tsv

```diff
@@ -10534,12 +10534,12 @@ ETC_20150428_011513	You need to kill the monsters to open the box!
 ETC_20150428_011514	No!
 ETC_20150428_011515	Activating the Altar
 ETC_20150428_011516	Unclean Insect Eggs
-ETC_20150428_011517	It can only be used in the Tomb Building Chapel.
+ETC_20150428_011517	It can only be used inside the Tomb Building Chapel.
 ETC_20150428_011518	Infrobirk Flame
 ETC_20150428_011519	Absorbed all the magical power.
 ETC_20150428_011520	Activate the Chapel Altar to kill the monsters!
 ETC_20150428_011521	I will open the door
-ETC_20150428_011522	Attacks are impossible because of the magic barrier
+ETC_20150428_011522	Attacks are ineffective because of the magic barrier
```

Comment:
Ineffective would mean the attacks are hitting but are not having a desired effect. Impossible is implying that attacks are not possible at all. Two different meanings, leave original translation